### PR TITLE
Mcping command

### PR DESCRIPTION
**Pull request by urielsalis**
Needs mcstatus installed(pip install mcstatus) and python

Assumes it will be run from same directory where the mcping command is(See Dockerfile)

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -11,8 +11,10 @@ RUN go install -v
 ###
 # Now generate our smaller image
 ###
-FROM alpine
+FROM jfloff/alpine-python
```
Comment (LordRalex) [inline Dockerfile]: Use an official image from alpine or python

File: handlers/mcping/mcping.go [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+
+func RunCommand(ds *discordgo.Session, mc *discordgo.MessageCreate, c *discordgo.Channel, cmd string, args []string) {
+	if len(args) == 0 {
+		return
+	}
+
+	server := args[0]
+
+	pythonCmd := exec.Command("python", "mcping.py", server)
+	out, err := pythonCmd.Output()
```
Comment (LordRalex) [inline handlers/mcping/mcping.go:17]: Should use CombinedOutput since this doesn't capture stderr

Comment (urielsalis) [reply handlers/mcping/mcping.go:17]: The script doesnt seem to use stderr, should I still use that?

File: Dockerfile (hunk 1 above)
Comment (urielsalis) [reply Dockerfile]: done

**Comment by randomairborne** [comment]
This PR is obsoleted by #23.

**Pull request closed without merging**